src/UploadedFile.php

```diff
@@ -280,7 +220,7 @@ public function moveTo($targetPath)
     /**
      * {@inheritdoc}
      *
-     * @return int|null The file size in bytes or null if unknown.
+     * @return int The file size in bytes or null if unknown.
```

Comment:
The phpdoc can be removed. I don't see why psr-7 made the size nullable. Is there a case when the uploaded file size is unknown? It is even set to 0 when no file was uploaded according to http://php.net/manual/en/features.file-upload.post-method.php

Reply:
I removed the phpdoc... I think I made this change because on constructor we do not allow to pass null. Should we?